src/main.tsx

```
@@ -1,9 +1,128 @@
+function getCSSVariable(name: string) {
+  return getComputedStyle(document.documentElement).getPropertyValue(name).trim();
+}
+
+// Helper with fallback
+function getCSSVariableOrFallback(name: string, fallback: string) {
+  const value = getCSSVariable(name);
+  return value || fallback;
+}
+
+// 2️⃣ Define app modes, toggle control, and palette
+function Main() {
```

ESLint

[error] Fast refresh only works when a file has exports. Move your component(s) to a separate file.

(react-refresh/only-export-components)